pkgs/top-level/aliases.nix

```diff
@@ -1372,6 +1372,7 @@ mapAliases ({
   trustedGrub-for-HP = throw "trustedGrub-for-HP has been removed, because it is not maintained upstream anymore"; # Added 2023-05-10
   tvbrowser-bin = tvbrowser; # Added 2023-03-02
   typst-fmt = typstfmt; # Added 2023-07-15
+  typst-preview = throw "The features of this program have been consolidated to 'tinymist', an all-in-one language server for typst"; # Added 2024-07-07
```

Comment:
This seems quite unclear: the user will only be prompted with just the message thrown, saying "this program" is abandoned, having no clues about which program "this program" actually is.

Reply:
On my side, running `nix build nixpkgs#typst-preview` shows the correct message:

```
❯ nix build nixpkgs#typst-preview
error:
       … while evaluating a branch condition

         at /nix/store/yzkx7gx2cn73dh3c7zg5rvgkp3yli1cd-source/pkgs/top-level/aliases.nix:33:5:

           32|   removeDistribute = alias: with lib;
           33|     if isDerivation alias then
             |     ^
           34|       dontDistribute alias

       … while evaluating a branch condition

         at /nix/store/yzkx7gx2cn73dh3c7zg5rvgkp3yli1cd-source/pkgs/top-level/aliases.nix:26:5:

           25|   removeRecurseForDerivations = alias: with lib;
           26|     if alias.recurseForDerivations or false
             |     ^
           27|     then removeAttrs alias [ "recurseForDerivations" ]

       (stack trace truncated; use '--show-trace' to show the full trace)

       error: The features of this program have been consolidated to 'tinymist', an all-in-one language server for typst
```

Reply:
@GaetanLepage I had to full text search this repo on github to find out it was for `typst-preview`, so it's probably best to make it "The features of ~this program~typst-preview have been consolidated..."

Reply:
Done !